protocol.html

```diff
@@ -717,6 +719,51 @@ <h4 property="schema:name"><span>Note</span>: Conditional Update</h4>
 
                   <p><span about="" id="server-etag" rel="spec:requirement" resource="#server-etag"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MAY">MAY</span> use the HTTP <code>ETag</code> header with a strong validator for RDF bearing representations in order to encourage clients to opt-in to using the <code>If-Match</code> header in their requests.</span></span></p>
                 </div>
+
+                <section id="n3-patch" inlist="" rel="schema:hasPart" resource="#n3-patch">
+                  <h4 property="schema:name">Modifying Resources Using N3 Patches</h4>
+                  <div datatype="rdf:HTML" property="schema:description">
+                    <p><span about="" id="server-patch-n3" rel="spec:requirement" resource="#server-patch-n3"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> support a <code>PATCH</code> request with an <em>N3 Patch</em> body when its target is an <em>RDF document</em> [<cite><a class="bibref" href="#bib-rdf11-concepts">RDF11-CONCEPTS</a></cite>], and indicate such support by listing <code>text/n3</code> as a value of the <code>Accept-Patch</code> header [<cite><a class="bibref" href="#bib-rfc5789">RFC5789</a></cite>] of relevant responses.</span></span> [<a href="https://github.com/solid/specification/issues/125#issuecomment-959518598" rel="cito:citesAsSourceDocument">Source</a>]</p>
+
+                    <p>An <em>N3 Patch</em> is a document in the <em>Notation3 (N3)</em> format [<cite><a class="bibref" href="#bib-notation3">N3</a></cite>], identified by the media type <code>text/n3</code>, conforming to the following structural constraints:</p>
+
+                    <ul>
+                      <li>The patch document can contain patch resources identified by a URI <code>?patch</code> which can be chosen freely.</li>
+                      <li>A patch resource may contain a triple <code>?patch rdf:type solid:Patch</code>.</li>
+                      <li>A patch resource contains at most one triple of the form <code>?patch solid:deletes ?deletions</code>.</li>
+                      <li>A patch resource contains at most one triple of the form <code>?patch solid:inserts ?insertions</code>.</li>
+                      <li>A patch resource contains at most one triple of the form <code>?patch solid:where ?conditions</code>.</li>
+                      <li>When present, <code>?deletions</code>, <code>?insertions</code>, and <code>?conditions</code> must be non-nested <em>cited formulae</em> [<cite><a class="bibref" href="#bib-notation3">N3</a></cite>] consisting only of triples and/or triple patterns. When not present, they are presumed to be the empty formula <code>{}</code>.</li>
```

Comment:
This section introduces the first occurrence of "triple pattern" in the protocol document. I think it would be useful to provide a reference link, or include it as a definition (might also be worth adding a definition for triple while we're at it).

Reply:
@csarven What is the best place for this?

Reply:
(I had hoped the Notation3 spec would define this, but it does not, because of quantification. SPARQL does, but it might not be entirely correct to cross-reference that here.)

Reply:
I'll need too leave this one open now, as "triple" is also used but not referenced in the document at the moment, so that's a dependency.

Reply:
Following up in https://github.com/solid/specification/issues/348